libs/pinetime_boot/src/display.c

```diff
@@ -215,7 +216,7 @@ static int init_display(void) {
     rc = hal_gpio_init_out(DISPLAY_CS, 1); assert(rc == 0);
     rc = hal_gpio_init_out(DISPLAY_DC, 0); assert(rc == 0);
     //  Switch on backlight
-    rc = hal_gpio_init_out(DISPLAY_HIGH, 0); assert(rc == 0);
+    rc = hal_gpio_init_out(DISPLAY_LOW, 0); assert(rc == 0);
```

Comment:
Do we need to set the other pins high explicitly? Or is this enough to guarantee low backlight only

Reply:
I *guess* they are high by default since the MEDIUM and LOW were not previously set.
But I can set them to high to remove any doubt.

Reply:
Ah I haven't looked around the bootloader code before, so if the pins are explicitly high by default that's fine (I have no clue what the default values are / where they come from)
Since it's free though, I think we may as well make it completely clear they're high

Reply:
To be honest, I'm not the original author of the firmware, and I barely know MyNewt so... that's another reason why I try to change as little code as possible :sweat_smile:

Reply:
Done :)